### PR TITLE
Send button on account

### PR DESCRIPTION
**Pull request by Tbaut**
- closes https://github.com/polkadot-js/apps/issues/1182

Takes all the heavy lifting done in the Transfer app. I didn't implement any overview of the accounts. What matters is the recipient’s address and identicon, and we have them already.

Propose a little lifting to the original transfer modal (without lifting), before / after
image

image

### CODE REVIEW

**Comment by jacogr** [comment]
Yes, the second works. However it is just the fees (imho), that needs some movement.

File: packages/app-accounts/src/Account.tsx [hunk 1]
```diff
@@ -110,7 +113,8 @@ class Account extends React.PureComponent<Props> {
     this.state = {
       isBackupOpen: false,
       isForgetOpen: false,
-      isPasswordOpen: false
+      isPasswordOpen: false,
+      isTransferOpen: false
```
Comment (jacogr) [inline packages/app-accounts/src/Account.tsx]: As a first step, think this is fine - but we probably need to give some thought of moving it elsewhere so it is re-usable. (Especially when we get to deposit on addresses - but for now, 100% - let's not try to complicate things, moving code is easy, getting it working initially is the hard bit)

**Comment by Tbaut** [comment]
Any suggestions for the fees? My last commits have the lifting from the second image you can take a look, it's rather.. hacky though :)

**Comment by jacogr** [comment]
Made some enhancements as a PR into this one - use, don't use - https://github.com/polkadot-js/apps/pull/1196

**Comment by Tbaut** [comment]
Here is how things look like now:
image

File: packages/app-accounts/src/modals/Transfer.tsx [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+  label.with-help {
+    flex-basis: 10rem;
+  }
+
+  .ui--Labelled-content {
+    flex: initial;
+    width: 40em;
+  }
+
+
```
[the remark this answers is not in the document]
Comment (Tbaut) [reply packages/app-accounts/src/modals/Transfer.tsx]: oops, thanks.

**Pull request merged**